bundles/org.openhab.binding.chromecast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -200,13 +214,23 @@
 		<item-type>Number:Time</item-type>
 		<label>Current Time</label>
 		<description>Current time of currently playing media</description>
+		<category>Time</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>Duration</tag>
```

Comment:
This seems to violate rule 3 in #12262, i.e. no property tag expected with the Status point tag.

Reply:
For the example temperature i can see it is not expected. But a song/video has a fixed length and this shows the progress (its status), you don't actively measure the progress. So i thought this was a good fit.

Reply:
That might be the case. I can't assess this myself since I can't find proper documentation and my knowledge about how these tags are used by the semantic model in MainUI is almost non-existant. @lolodomo, perhaps you can help here? 

@lsiepel - maybe let me ask to get this going, when adding these tags, which effect do you see in MainUI, rules or elsewhere, i.e. what is gained from this change?

Reply:
> Are you asking why there are semantic tags at all?

No, I'm asking if you can explain or show a screenshot of the difference that can be seen when having the `Duration` tag compared to when not having it.